.env.template

```diff
@@ -1,5 +1,6 @@
 export APP_SETTINGS="config.DevelopmentConfig"
 export DATABASE_URL="postgresql://localhost/freefrom_map_dev"
+export CMS_URL="http://freefrom-cms.nitro"
```

Comment:
This is the local URL of the CMS. I'm still cleaning up the CMS repo and writing a good README for it, but I'll be sure to share that around once it's done.